lib/active_model/serializer/error_serializer.rb

```diff
@@ -0,0 +1,10 @@
+class ActiveModel::Serializer::ErrorSerializer < ActiveModel::Serializer
+  # @return [Hash<field_name,Array<error_message>>]
+  def as_json
+    object.errors.messages
+  end
```

Comment:
It doesn't really need to inherit from serializer at this point.. but for interface purposes I am